mmv1/templates/terraform/custom_import/compute_instance_group_membership.go.erb

```diff
@@ -0,0 +1,19 @@
+config := meta.(*transport_tpg.Config)
+
+if err := tpgresource.ParseImportId([]string{
+    "projects/(?P<project>[^/]+)/regions/(?P<zone>[^/]+)/instanceGroups/(?P<instance_group>[^/]+)/(?P<instance>.+)",
+    "(?P<project>[^/]+)/(?P<zone>[^/]+)/(?P<instance_group>[^/]+)/(?P<instance>.+)",
+    "(?P<zone>[^/]+)/(?P<instance_group>[^/]+)/(?P<instance>.+)",
+    "(?P<instance_group>[^/]+)/(?P<instance>.+)",
```

Comment:
Similar to a previous comment, I'm unclear if the zone can be omitted for an import

Reply:
For the custom import, it's created based on the auto-generated imports and only added instance in the end to be able to parse the resource

Reply:
Ok yea, makes sense. I think this variant was generated because the zone is in fact optional (because it can be specified at the provider level).